### ISSUE TITLE
Output under bindgen::EnumVariation::ModuleConsts missing ::Type?

### ISSUE THREAD

**Issue by jcreedcmu** [rust-lang-nursery/rust-bindgen#1418]
### Input C/C++ Header

```C++
// header.h
typedef enum { bar, baz, blap } Foo;
void func(Foo x);
```

### Bindgen Invocation

```Rust
bindgen::Builder::default()
        .header("header.h")
        .default_enum_style(bindgen::EnumVariation::ModuleConsts)
        .generate()
        .unwrap();
```

### Actual Results

```rust
/* automatically generated by rust-bindgen */

pub mod Foo {
    pub type Type = u32;
    pub const bar: Type = 0;
    pub const baz: Type = 1;
    pub const blap: Type = 2;
}
extern "C" {
    pub fn func(x: Foo);
}
```

When I put 
```
include!(concat!(env!("OUT_DIR"), "/bindings.rs"));

fn main() {
    println!("Hello, world!");
}
```
in `main.rs` and try to build, I get
```
error[E0573]: expected type, found module `Foo`
  --> /tmp/bindgensb/target/debug/build/bindgensb-778298c9127d6c38/out/bindings.rs:10:20
   |
10 |     pub fn func(x: Foo);
   |                    ^^^ not a type
```

### Expected Results

```rust
/* automatically generated by rust-bindgen */

pub mod Foo {
    pub type Type = u32;
    pub const bar: Type = 0;
    pub const baz: Type = 1;
    pub const blap: Type = 2;
}
extern "C" {
    pub fn func(x: Foo::Type);
}
```

**Comment by emilio** [comment]
Looks like a `rustfmt` bug? At least master generates the expected code here.

**Comment by jcreedcmu** [comment]
Hmm, if I throw in
`.rustfmt_bindings(false)` I still get 
`pub mod Foo { pub type Type = u32 ; pub const bar : Type = 0 ; pub const baz : Type = 1 ; pub const blap : Type = 2 ; } extern "C" { pub fn func ( x : Foo ) ; }`

My `Cargo.toml` has
```
[build-dependencies]
bindgen = { git = "https://github.com/rust-lang-nursery/rust-bindgen.git", rev = "988a9" }
```

Any other ideas what I might be doing wrong?

**Comment by jcreedcmu** [comment]
Should I be paranoid and suspect the rust toolchain version matters? I'm on `rustc 1.30.0-nightly (0198a1ea4 2018-09-08)`.

**Comment by emilio** [comment]
Rust toolchain shouldn't matter. I tested with the CLI using `--constified-enum-module Foo` on that header, but there may be something related to it being the default. Let me try again :)

**Comment by jcreedcmu** [comment]
Yeah, didn't think so. Oh, I should note that I also observe `--constified-enum-module Foo` works just fine --- it's _only_ the `build.rs` with `bindgen::EnumVariation::ModuleConsts` that causes this error for me!

**Comment by emilio** [comment]
Ok, yeah, I can repro with:

```sh
$ ./target/debug/bindgen t.h --default-enum-style moduleconsts
```

Should be trivial to fix, thanks for the catch!

**Comment by jcreedcmu** [comment]
Oh nice, I entirely missed that `--default-enum-style` was an option at all from the cli because I just grepped `bindgen --help` for `'module'`. Anyway, thanks!

**Comment by emilio** [comment]
Fix is https://github.com/rust-lang-nursery/rust-bindgen/pull/1419
